src/tests/EdFi.DataManagementService.Tests.E2E/Features/RESTAPIConventions/NoAuthRESTAPIConventions.feature.disabled

```diff
@@ -0,0 +1,232 @@
+Feature: No Authenticated - No Authorized REST API Conventions
+
+    Rule: Not Authenticated
+
+        Background:
+            Given the user is not authenticated
+
+        @ignore
+        Scenario: 01 Verify user cannot retrieve a resource being unauthenticated
+             When a GET request is made to "/ed-fi/students"
+             Then it should respond with 401
+              And the response body is
+                  """
+                  {
+                      "detail": "The caller could not be authenticated.",
+                      "type": "urn:ed-fi:api:security:authentication",
+                      "title": "Authentication Failed",
+                      "status": 401,
+                      "correlationId": null,
+                      "errors": [
+                          "Authorization header is missing."
+                      ]
+                      }
```

Comment:
Minor thing, format the curly brackets for the json.

Reply:
Changed